oioioi/quizzes/controllers.py

```diff
@@ -222,6 +222,8 @@ def render_report_failure(self, request, report):
         return ProblemController.render_report(self, request, report)
 
     def render_report(self, request, report):
+        if report.kind == u'FAILURE':
+            return super(QuizProblemController, self).render_report(request, report)
         problem_instance = report.submission.problem_instance
         if report.kind == 'FAILURE':
```

Comment:
we have the same if here wit ha different action - are you sure this commit wasn't upstreamed previously, and then mis-rebased?

Reply:
Indeed, I will remove that commit.